Source/Core/Core/ConfigManager.h

```diff
@@ -53,91 +53,93 @@ struct SConfig : NonCopyable
   bool m_RecursiveISOFolder;
 
   // Settings
-  bool bEnableDebugging;
+  bool bEnableDebugging = false;
 #ifdef USE_GDBSTUB
   int iGDBPort;
 #ifndef _WIN32
   std::string gdb_socket;
 #endif
 #endif
-  bool bAutomaticStart;
-  bool bBootToPause;
+  bool bAutomaticStart = false;
+  bool bBootToPause = false;
 
   int iCPUCore;
 
   // JIT (shared between JIT and JITIL)
-  bool bJITNoBlockCache, bJITNoBlockLinking;
-  bool bJITOff;
-  bool bJITLoadStoreOff, bJITLoadStorelXzOff, bJITLoadStorelwzOff, bJITLoadStorelbzxOff;
-  bool bJITLoadStoreFloatingOff;
-  bool bJITLoadStorePairedOff;
-  bool bJITFloatingPointOff;
-  bool bJITIntegerOff;
-  bool bJITPairedOff;
-  bool bJITSystemRegistersOff;
-  bool bJITBranchOff;
-  bool bJITILTimeProfiling;
-  bool bJITILOutputIR;
+  bool bJITNoBlockCache = false, bJITNoBlockLinking = false;
```

Comment:
You may want to put these variable declarations on their own line, just to make them nicer to read vertically. Ditto for below occurrences. If you don't want to, that's fine as well.

Other than this, it all looks good to me; thanks for doing this =)